plugin/pkg/scheduler/scheduler.go

```diff
@@ -107,6 +107,7 @@ func New(c *Config) *Scheduler {
 
 // Run begins watching and scheduling. It starts a goroutine and returns immediately.
 func (s *Scheduler) Run() {
+	go metrics.ReportBindingRateLimiterSaturation(s.config.BindPodsRateLimiter)
```

Comment:
Perhaps add a name to rate-limiter creation, and have rate limiters auto-register themselves?